README.rst

```diff
@@ -20,7 +20,9 @@ Description
 
 A tool for estimating growth rates in growth curves. The tool fits λ ⋅ e :sup:`μ⋅x` + N :sub:`0` to any candidate growth phases of the growth curve that have increasing growth, i.e. where both the first and second derivative of the growth function are positive. To identify these phases reliably, the tool utilizes a custom smoothing function that addresses problems other smoothing methods have with growth curves that have regions with varying levels of noise (e.g. lots of noise in the beginning, then less noise after growth starts, then more noise in the stationary phase). 
 
-The parameter N :sub:`0` of the model can optionally be constrained. This is recommended if the value is known. The growth rate in calculated growth phases can only be properly compared if their N :sub:`0` (baseline OD; when the organism is at its initial population) points to a similar stage of actual growth.
+The parameter N :sub:`0` is - contrary to common nomenclature - represents the background/blank OD reading and can optionally be constrained. This is recommended if the value is known.
+
+The growth rate in calculated growth phases can only be properly compared if their λ (seeding OD; when the organism is at its initial population) points to a similar stage of actual growth. Currently λ reported by this package represents ??? and should not be used as a comparison measure.
```

Comment:
The λ represents the y-axis intercept. The value originally referred to the lag time (λ in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC184525/?page=2). Since the exponential model we use now can not measure lag time, the value is meaningless.